packages/react-native/src/Authenticator/common/DefaultContent/DefaultContent.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { Fragment, useMemo } from 'react';
+import { View } from 'react-native';
+
+import { Button, ErrorMessage } from '../../../primitives';
+import { TextFieldOptionsType, RadioFieldOptions } from '../../hooks';
+import { StrictTheme, useTheme } from '../../../theme';
+
+import { getDefaultStyle } from './styles';
+import { DefaultContentProps } from './types';
+
+function useThemedStyles<Style>(getStyle: (theme: StrictTheme) => Style) {
+  const theme = useTheme();
+  return useMemo(() => getStyle(theme), [getStyle, theme]);
+}
```

Comment:
Will move this to this to live with theming in a follow up, needs to be able to handle additional params